cmd/kaniko-docker/main.go

```diff
@@ -460,6 +465,7 @@ func run(c *cli.Context) error {
 			SourceTarPath:               c.String("source-tar-path"),
 			UseNewRun:                   c.Bool("use-new-run"),
 			IgnorePath:                  c.String("ignore-path"),
+			IgnorePaths:                 c.StringSlice("ignore-paths"),
```

Comment:
What's the delimiter? Apparently commas are legal in some unix/windows systems. I haven't tested. So make sure we use a delimiter that's not allowed (eg: a semi-colon?)

Reply:
Commas are not valid in windows. StringSlice is the common thing to handle path across. This should be safe

Reply:
they are allowed at one point. But it's usually considered as not valid in programming world to avoid confusing. At one point most ASCIIs are allowed in windows file path and that was deprecated